Project.toml

```
@@ -31,11 +32,12 @@ test = ["DataStructures", "DataValues", "Dates", "LaTeXStrings", "Random", "Test
 
 [compat]
 julia = "1"
-Missings = ">= 0.2.3"
 CategoricalArrays = ">= 0.5.4"
-StatsBase = ">= 0.11.0"
 Compat = ">= 0.59.0"
-Tables = ">= 0.2.3"
+InvertedIndices = ">= 1.0.0"
```

This should be OK since the package has reached 1.0:
```suggestion
InvertedIndices = "1"
```